packages/maskbook/src/utils/hooks/useRemoteControlledDialog.ts

```diff
@@ -50,3 +50,23 @@ export function useRemoteControlledDialog<T extends { open: boolean }>(
     )
     return [open, onUpdateByLocal] as const
 }
+
+export function useRemoteControlledDialogEvent(event: UnboundedRegistry<{ open: boolean }>) {
```

Comment:
```suggestion
export function useRemoteControlledDialogEvent<T extends { open: boolean }>(event: UnboundedRegistry<T>) {
```

Reply:
Fixed in https://github.com/DimensionDev/Maskbook/pull/2853/commits/753d4871e25940716ce3362b7922277cd080a557. BTW, this will be removed in favor of https://github.com/DimensionDev/Maskbook/pull/2916